journal-parsing/src/test/resources/samle-receivers-reset.csv

```diff
@@ -0,0 +1,26 @@
+"JOURNAL_RECEIVER_NAME","FIRST_SEQUENCE_NUMBER","LAST_SEQUENCE_NUMBER","ATTACH_TIMESTAMP","DETACH_TIMESTAMP","SAVE_TIMESTAMP","PREVIOUS_JOURNAL_RECEIVER","NEXT_JOURNAL_RECEIVER"
```

Comment:
Note the typo in the filename for both this and the other CSV.

Are these actually used in any tests? I couldn't see any references the file name in this pull request.

Reply:
deleted